consumer_test.go

```diff
@@ -33,6 +33,14 @@ var _ = Describe("Consumer", func() {
 		}()
 	}
 
+	It("should fail for invalid group id", func() {
+		_, err := newConsumer("")
+		Expect(err).To(HaveOccurred())
+
+		_, err = newConsumer("foo:bar")
```

Comment:
That's an interesting one. I just checked and it seems consumer groups may contain colons too